plugins/endpoint.ts

```diff
@@ -0,0 +1,9 @@
+import '@polkadot/api-augment'
+import Connector from '@vue-polkadot/vue-api'
+
+export default ({ store }) => {
+  const endpoint = store.state.setting.apiUrl
```

Comment:
so because vuex-persist plugin is being loaded beforehand we can always be assured that `setting.apiUrl` is set right?

Reply:
It was problem that vuex persist used older approach and that the session storage was rehydrated too late